semantic_search/ncbi.py

```diff
@@ -21,7 +21,6 @@ def _compact(input: List) -> List:
 
 # -- Setup and initialization --
 MAX_EFETCH_RETMAX = 10000
-MAX_LIST_RETMAX = 100000
```

Comment:
Did you mean to change this?

Reply:
Just address this then LGTM!

Reply:
Yeah, we don't need that environment variable. It was used in the extra method I made for 'pubmed-dl'.